modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/resources/META-INF/resources/js/CKEditor5ReactClassicEditor.tsx

```diff
@@ -12,8 +12,10 @@ import React from 'react';
 
 const CKEditor5ReactClassicEditor = ({
 	editorConfig,
+	editorTransformerURLs,
 }: {
 	editorConfig: ClassicEditorConfig;
+	editorTransformerURLs?: Array<string>;
```

Comment:
In previous versions of CKEditor, we put the `editorTransformerURLs` as a new entry inside `editorConfig` object. This is the only choice we have [when injecting configuration from the back end.](https://github.com/liferay/liferay-portal/blob/058415b50209587c7e3f03eb2824b9e314a7e9c4/modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/portlet/editor/config/contributor/CETEditorConfigContributor.java#L71-L85)

If we separate things here, perhaps it's ok for the sample, but when running the CX out of the sample, I'm afraid it won't work.

Reply:
Oh wait, that file belongs to the sample. So, what I've said does not apply. Here in the sample, `editorTransformerURLs` is passed from the context. So it's fine. [This](https://github.com/liferay-frontend/liferay-portal/pull/4898/commits/2bb4b60aef51f0e7ca11bbb9d21d0a1e0f5368f3#diff-c1be30a326ba5a5fa28265f0fa67d5b88bf081d8a668b87ba0fda1241149beffR129) is what matters outside the sample.